app/services/embedding_service.py

```
@@ -0,0 +1,248 @@
+            '지성': '지성피부',
+            '복합성': '복합성피부'
+        }
+        
+        for old, new in replacements.items():
+            text = text.replace(old, new)
+        
+        return text
+    
+    @classmethod
+    def preprocess_product_text(cls, product_name: str, brand: str, 
+                              description: str, ingredients: str = None,
```

Flake8 (7.2.0)

[error] continuation line under-indented for visual indent

(E128)